### DISCUSSION TITLE
Support for nested drop downs is much needed!

### DISCUSSION THREAD

**Discussion by NielsKSchjoedt** [getredash/redash#6744] [Ideas]
_This is a copy of: https://discuss.redash.io/t/observable-query-based-dropdown-list/3229_

Redash is such a great tool 🎉 - however, one thing I really lack is nested query based drop downs. Basically the idea is, that if your database has two columns, say: `country, city` then those could be used to populate two nested drop downs. One for country, one for city. However currently, that is not possible because the queries powering drop downs cannot interdepend:

![Skærmbillede 2024-02-05 kl  11 57 56](https://github.com/getredash/redash/assets/570755/79ef4cfb-9499-4ab5-8354-a343de1d41cf)

**Example of what we need**

You could have a query like:

```sql
SELECT DISTINCT country
FROM my_db.my_table 
ORDER BY country
```

Then add a "Query Based Drop Down" with those countries

Then use the selected variable in another drop down query for city:

```sql
SELECT DISTINCT city
FROM my_db.my_table 
WHERE country = '{{country}}'
ORDER BY city
```

It is such a pain this functionality does not exist 😬

**Comment by sonnguyenxaion** [comment]
Any update?????
